### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by borkarsaish65**
image

Added Modal implementation.

Demo can be checked from here -> https://voluble-pika-55d3b0.netlify.app/component/modal-component/modal.html

### CODE REVIEW

**Review by Abhijeet43 — APPROVED** [review]

**Review by Abhijeet43 — APPROVED** [review]

File: component/button/button.css [hunk 1]
```diff
@@ -18,6 +18,18 @@
...
+.btn-danger {
+    background-color: rgb(255, 62, 62);
+    color:white;
+}
+
+.btn.btn-primary:hover{
+    background-color: #1b449b;
+}
+.btn.btn-danger:hover{
+    background-color: rgb(174 37 37);;
```
Comment (DevRaahul) [inline component/button/button.css:30]: remove extra colon @line 30

File: component/modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    position: absolute;
+    background-color: white;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    max-width: 25rem;
+    border:1px solid rgb(0 0 0 / 39%);
+    padding:20px;
+    border-radius: 5px;
+}
```
Comment (DevRaahul) [inline component/modal/modal.css:16]: Suggestion : leave one empty line after each class this will your code more readable. Use color constant

**Review by DevRaahul — APPROVED** [review]

File: component/button/button.css (hunk 1 above)
Comment (borkarsaish65) [reply component/button/button.css:30]: wow nice catch. will do sir.

File: component/modal/modal.css (hunk 2 above)
Comment (borkarsaish65) [reply component/modal/modal.css:16]: ok sure. Thanks

**Pull request merged**